ups/eupspkg.cfg.sh

```diff
@@ -1,7 +1,93 @@
 export SCONSFLAGS=$SCONSFLAGS" USE_UNKNOWN_VARS=true TMV_DIR="$TMV_DIR" PREFIX="$PREFIX" PYPREFIX="$PREFIX"/lib/python EXTRA_LIB_PATH="$TMV_DIR"/lib EXTRA_INCLUDE_PATH="$TMV_DIR"/include"
 
 pathToPython=$(which python)
-export DYLD_FALLBACK_LIBRARY_PATH="${pathToPython%bin/python}/lib"
+pathToPythonLib="${pathToPython%bin/python}/lib"
```

Comment:
This is actually remarkably flaky looking at my system's homebrew Python. Luckily we have code later on that only tries to do things if we are not in `/usr`. Miniconda3 does seem to have a python library in `../lib` but it's not actually called `libpython3.5.dylib`...